### ISSUE TITLE
sphinx.mo permission error

### ISSUE THREAD

**Issue by wwguo** [sphinx-doc/sphinx#2522]
I encounter with the following error. 

```
# Sphinx version: 1.4.1
# Python version: 3.5.1 (CPython)
# Docutils version: 0.12 release
# Jinja2 version: 2.8
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/sphinx/cmdline.py", line 243, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/usr/lib/python3.5/site-packages/sphinx/application.py", line 189, in __init__
    self._init_i18n()
  File "/usr/lib/python3.5/site-packages/sphinx/application.py", line 214, in _init_i18n
    charset=self.config.source_encoding)
  File "/usr/lib/python3.5/site-packages/sphinx/locale/__init__.py", line 217, in init
    catinfo.write_mo(language)
  File "/usr/lib/python3.5/site-packages/sphinx/util/i18n.py", line 58, in write_mo
    with io.open(self.mo_path, 'wb') as mo:
PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.5/site-packages/sphinx/locale/de/LC_MESSAGES/sphinx.mo'
```

Here is the permission of files.

```
$ ls -l
drwxr-xr-x 3 root root    80 9月  19 2015 de

$ ls -l de/
drwxr-xr-x 2 root root 144 4月  18 13:25 LC_MESSAGES

$ ls -l de/LC_MESSAGES/
-rw-r--r-- 1 root root  4050 4月  12 21:07 sphinx.js
-rw-r--r-- 1 root root 11054 4月  12 21:07 sphinx.mo
-rw-r--r-- 1 root root 21595 4月  12 21:10 sphinx.po
```

**Comment by kiwifb** [comment]
I was wanting to open a similar issue here. We encountered a similar problem in sage-on-gentoo in issue https://github.com/cschwan/sage-on-gentoo/issues/416. My diagnostic is that the system folder for .mo is not skipped when it should. My solution is https://github.com/cschwan/sage-on-gentoo/blob/33f4d5e220006f1a4848dd40a1d75d47838d1211/dev-python/sphinx/files/sphinx-1.4.1-locale_dir.patch
but I am worried it could have side effect.

**Comment by shimizukawa** [comment]
sphinx.po under installed sphinx/locale directory have to be excluded from `compile_catalogs` target. I'll fix it.

**Comment by kiwifb** [comment]
I see, so my fix is overly broad, which is something I was afraid of.

**Comment by shimizukawa** [comment]
Fixed. @wwguo @kiwifb please check a changed behavior with latest stable code if you have chance.

**Comment by kiwifb** [comment]
Fixes the issue for me.

**Comment by shimizukawa** [comment]
@kiwifb Thanks!

**Comment by wwguo** [comment]
Fixes for me too. Thanks!

**Comment by jandicke** [comment]
I'm new to Python and Sphinx, and it's possible I did something wrong. Yesterday I installed Sphinx on my Macbook Pro (OS X 10.11.5). Trying to build my first document ended up with:

```
# Sphinx version: 1.4.1
# Python version: 2.7.11 (CPython)
# Docutils version: 0.12 release
# Jinja2 version: 2.8
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/cmdline.py", line 243, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/application.py", line 189, in __init__
    self._init_i18n()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/application.py", line 214, in _init_i18n
    charset=self.config.source_encoding)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/locale/__init__.py", line 217, in init
    catinfo.write_mo(language)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/util/i18n.py", line 58, in write_mo
    with io.open(self.mo_path, 'wb') as mo:
IOError: [Errno 13] Permission denied: '/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/sphinx/locale/de/LC_MESSAGES/sphinx.mo'
```

So I replaced the three files mentioned above with the code I got when I moved to https://github.com/sphinx-doc/sphinx/commit/26fecafb4920a8d9dacaa9e3614bf182ce12696a and copied and pasted the source code of the three files in view mode.

Now I get another error:

```
# Sphinx version: 1.4.1
# Python version: 2.7.11 (CPython)
# Docutils version: 0.12 release
# Jinja2 version: 2.8
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2$
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2$
    self.config.pre_init_values(self.warn)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2$
    raise AttributeError('No such config value: %s' % name)
AttributeError: No such config value: pre_init_values
```

May anybody be so kind to help me? I'm really interested in using Sphinx. :)

Thanks,

Jan

**Comment by tk0miya** [comment]
This has fixed at Sphinx-1.4.2. It seems version of sphinx you used is 1.4.1.
Could you update it to 1.4.2 and try again?

**Comment by jandicke** [comment]
Thanks a lot! I don't know what exactly went wrong. Maybe Macport executed an autoupdate I dIdn't recognize, and I updated the 1.4.2 version files by mistake. Nevertheless: I deleted and installed the system again, and I got a working 1.4.2 version. :)
